GLTFSDK/Source/Schema.cpp

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <GLTFSDK/Schema.h>
+#include <GLTFSDK/Exceptions.h>
+
+#include "SchemaJson.h" // Auto-generated header, don't include in any other translation units to avoid linker errors
+
+using namespace Microsoft::glTF;
+
+namespace
+{
+    class DefaultSchemaLocator : public ISchemaLocator
+    {
+    public:
+        const char* GetSchemaContent(const std::string& uri) const override
+        {
+            auto it = SchemaJson::GLTF_SCHEMA_MAP.find(uri);
+
+            if (it == SchemaJson::GLTF_SCHEMA_MAP.end())
+            {
+                throw GLTFException("Unknown Schema uri: " + uri);
+            }
+
+            return it->second.c_str();
+        }
+    };
+}
+
+// Microsoft::glTF namespace function definitions
```

Comment:
Unnecessary, we don't have this in all the other files.

Reply:
I've done something similar in .cpp files where there are defintions for multiple classes etc. (e.g. ExtensionsKHR.cpp).